src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('span.population');
```

Comment:
Ensure that there are elements with the class 'population' in the DOM. If there are none, this will result in an empty NodeList, and subsequent operations will not work as expected.